docs/faqs.md

```
@@ -56,6 +56,8 @@ Use the same commands to debug CLI scripts.
 
 To disable, run `ahoy up`.
 
+See https://www.vortextemplate.com/docs/tools/xdebug for more details.
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)